includes/class-wc-calypso-bridge-customize-store.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Adds Customize Store task related functionalities
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   1.0.0
+ * @version 2.2.14
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Calypso Bridge Customize Store
+ */
+class WC_Calypso_Bridge_Customize_Store {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Calypso_Bridge_Customize_Store instance
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor.
+	 */
+	private function __construct() {
+
+		// Only in Ecommerce or Free Trial
+		if ( ! wc_calypso_bridge_has_ecommerce_features() && ! wc_calypso_bridge_is_ecommerce_trial_plan() ) {
```

Comment:
How about checking the feature flag instead? I think it would keep us in sync with the planned rollout without needing to make changes in the future. The rollout currently looks like:

October: Woo Express only
December: Everywhere (Core, all WP.com hosted plans)

Between Oct-Dec, e-commerce plans could still be used for testing by changing the feature flag on the site.

Reply:
> wp-env run cli wp option get woocommerce_admin_customize_store_completed

I always forget we have access to CLI in WoA dev site :) Thank you for the tip 👍

Reply:
@adrianduffell Updated in https://github.com/Automattic/wc-calypso-bridge/pull/1307/commits/dee4b2537b577e1ec5c2ca204fcd9b3cbf058d8c

Could you run the test again, please? I've also updated the test instructions.

Reply:
Thanks @moon0326 -- I tested it again and it's all working nice!